packages/itinerary-body/src/AccessLegBody/mapillary-button.tsx

```diff
@@ -27,7 +27,8 @@ const generateBoundingBoxFromCoordinate = ({
   return [west, south, east, north];
 };
 
-const Container = styled.a`
+const Container = styled.a<{ hideDrivingDirections: boolean }>`
+  margin-left: ${props => (props.hideDrivingDirections ? ".4em" : "0")};
```

Comment:
To avoid passing a prop, add an after margin to the steps header span instead of setting a margin here.

Reply:
Good call, fixed in [53a5c5f](https://github.com/opentripplanner/otp-ui/pull/658/commits/53a5c5f9b01fdd1b78e0a02d13743805c600454b)